lib/widgets/compose_box.dart

```diff
@@ -210,6 +213,76 @@ class _StreamContentInputState extends State<_StreamContentInput> {
   }
 }
 
+/// Data on a file to be uploaded, from any source.
+///
+/// A convenience class to represent data from the generic file picker,
+/// the media library, and the camera, in a single form.
+class _File {
+  _File({required this.content, required this.length, required this.filename});
+
+  final Stream<List<int>> content;
+  final int length;
+  final String filename;
+}
+
+_uploadFiles({
```

Comment:
```suggestion
void _uploadFiles({
```

Otherwise its return type is `dynamic`.

(There's probably a lint for this, which we should enable. If we ever do want a function returning `dynamic`, or a variable of type `dynamic`, etc., I'd want to say so explicitly.)

Reply:
`Future<void>`, I think, since it's async?

Reply:
If we're going to await the result, then definitely `Future<void>` instead. To explicitly say it's not the caller's business whether this function completes synchronously or does some async work, one can say `void`.

The docs on this are very terse: https://dart.dev/language/built-in-types

Better description here, perhaps: https://www.reddit.com/r/dartlang/comments/oo65yd/does_void_mean_return_null_in_dart_or_simply_just/

Reply:
Ah OK, that makes sense. Since you pointed out that we should indeed await the result, in https://github.com/zulip/zulip-flutter/pull/70#discussion_r1173019126, I plan to go with `Future<void>`.